assets/js/show-trip-script.js

```diff
@@ -0,0 +1,32 @@
+// Control location photos in slides 
```

Comment:
can you add a small docstring at the top of the file like the one below, to describe what the File is doing? 
```
/**
 * [ description]
 */
```

Reply:
Sure, added in commit 82f56dd

Reply:
Thanks :+1: